Sources/SentryCrash/Installations/SentryCrashInstallation.h

```diff
@@ -43,12 +43,13 @@
  * WARNING: Only call async-safe functions from this function! DO NOT call
  * Objective-C methods!!!
  */
-@property (atomic, readwrite, assign) SentryCrashReportWriteCallback onCrash;
+@property (nullable, atomic, readwrite, assign) SentryCrashReportWriteCallback onCrash;
```

Comment:
`m`: Why did you add `nullable` here?

Reply:
<img width="1226" alt="Screenshot 2023-11-07 at 1 50 40 PM" src="https://github.com/getsentry/sentry-cocoa/assets/1187918/10cfc58b-990e-4c54-8f9f-2ced3a818281">
It seems by adding nullability specifies elsewhere in the file Xcode now requires it everywhere. I guessed that the value could be `nullable`, but I can change it to `_Null_unspecified`.

Reply:
That's a strange one, since `assign` isn't an object type. In any case, for this header, if you wind up needing to mark anything as `nullable`, could you instead use `NS_ASSUME_NONNULL_BEGIN`/`NS_ASSUME_NONNULL_END` and only mark the things that are `nullable`? (And, this property should not be `nullable`. Between this and https://github.com/getsentry/sentry-cocoa/pull/3369#discussion_r1385110224, it looks like you may not need this any more in this header.)

Reply:
Changed to use `NS_ASSUME_NONNULL_BEGIN` / `NS_ASSUME_NONNULL_END`